dashboard/src/components/common/Table.tsx

```diff
@@ -15,7 +60,12 @@ export default function Table({ data }: { data: { [key: string]: any }[] }) {
               scope='col'
               key={column}
             >
-              {column}
+              <HBox alignment={align.start}>
+                <IconContext.Provider value={{ className: 'inline-block', size: '1.5em' }}>
+                  {getIconForDataType(dataTypes?.[column])}
+                </IconContext.Provider>
+                <div className={'inline-block pl-3'}>{column}</div>
```

Comment:
nit: try it out but I think I'd reduce the left padding a bit, I feel too much distance between the icon and the text
```suggestion
                <div className={'inline-block pl-1'}>{column}</div>
```